benches/speed_tests.rs

```diff
@@ -188,63 +192,215 @@ pub fn count_vec(c: &mut Criterion) {
 /// Benchmark prio3 client performance.
 pub fn prio3_client(c: &mut Criterion) {
     let num_shares = 2;
+    let len = 1000;
```

Comment:
There's a lot of code movement here, which makes it very difficult to review carefully. Because there is a functional change here, would you try writing the change in a way that minimizes size of the diff?

Reply:
Removed our superfluous code changes, the diff should now be readable.